src/macaron/config/defaults.ini

```diff
@@ -278,7 +281,6 @@ interpreter =
 interpreter_flag =
     -m
 build_arg =
-    install
```

Comment:
Why do we remove `install` from this `build_arg` field?

Reply:
That's because of false positives. `pip install` is often used for installing dependencies, which shouldn't be considered as a build command for the main package.